test/org/sagebionetworks/bridge/models/schedules/CriteriaScheduleStrategyTest.java

```diff
@@ -260,55 +264,32 @@ public void validatesInvalidScheduleCriteria() {
         setUpStrategyWithAppVersions();
         setUpStrategyWithOneRequiredDataGroup();
         setUpStrategyWithProhibitedDataGroups();
-        setUpStrategyNoCriteria();
+        setUpStrategyEmptyCriteria();
         
+        Criteria criteria = Criteria.create(-2, -10, null, null);
         // We're looking here specifically for errors generated by the strategy, not the plan
         // it's embedded in or the schedules. I've made those valid so they don't add errors.
-        strategy.addCriteria(new ScheduleCriteria.Builder()
-                .withMinAppVersion(-2)
-                .withMaxAppVersion(-10)
-                .withSchedule(SCHEDULE_FOR_STRATEGY_WITH_ONE_REQUIRED_DATA_GROUP).build());
-        
-        // Has no schedule
-        strategy.addCriteria(new ScheduleCriteria.Builder().build());
+        strategy.addCriteria(new ScheduleCriteria(SCHEDULE_FOR_STRATEGY_WITH_ONE_REQUIRED_DATA_GROUP, criteria));
```

Comment:
We check with both validation and preconditions that a ScheduleCriteria object has a schedule and a criteria object, and both of those are validated. Added validation for the collection of scheduleCriteria objects being completely absent (which was missing).